modules/metrics/README.md

```diff
@@ -203,6 +204,22 @@ ffwd.discovery = {
 }
 ```
 
+### duration-threshold-metrics
+
+key | type | required | note
+--- | ---- | -------- | ----
+`duration-threshold-metrics.{endpoint}.{method}` | int | required | Duration threshold in milliseconds for an endpoint/method combination
+
+#### Example
+
+```
+duration-threshold-metrics = {
+  /v1/test-a.GET = 200 # meter marked every time this endpoint resolves in under 200 ms
+  /v1/test-b.GET = 100 # meter marked every time this endpoint resolves in under 100 ms
```

Comment:
Can you add an example on how the final metrics would look like under the `Metrics` section?